pkg/registry/generic/registry/store.go

```diff
@@ -45,94 +45,112 @@ import (
 	"github.com/golang/glog"
 )
 
-// Store implements pkg/api/rest.StandardStorage.
-// It's intended to be embeddable, so that you can implement any
-// non-generic functions if needed.
-// You must supply a value for every field below before use; these are
-// left public as it's meant to be overridable if need be.
-// This object is intended to be copyable so that it can be used in
-// different ways but share the same underlying behavior.
+// Store implements pkg/api/rest.StandardStorage. It's intended to be
+// embeddable, so that you can implement any non-generic functions if needed.
+// You must supply a value for every field below before use; these are left
```

Comment:
This sentence is confusing. They must be set or they are optional? 

"Must be non-nil" ?